docs/operating-scylla/nodetool-commands/tasks/list.rst

```diff
@@ -42,21 +42,21 @@ For single list:
 
 .. code-block:: shell
 
-    task_id                              type   kind scope    state sequence_number keyspace table entity
-    5116ddb6-85b5-4c3e-94fb-72128f15d7b4 repair node keyspace done  3               abc
+    task_id                              type   kind scope    state sequence_number keyspace table entity shard start_time           end_time
+    5116ddb6-85b5-4c3e-94fb-72128f15d7b4 repair node keyspace done  3               abc                   0     2025-01-16T16:12:11Z 2025-01-16T16:12:13Z
 
 With repetition:
 
 .. code-block:: shell
 
-    task_id                              type   kind scope    state sequence_number keyspace table entity
-    d8926ee7-0faf-47b7-bfeb-82477e0c7b33 repair node keyspace done  5               abc
-    1e028cb8-31a3-45ed-8728-af7a1ab586f6 repair node keyspace done  4               abc
+    task_id                              type   kind scope    state sequence_number keyspace table entity shard start_time           end_time
+    d8926ee7-0faf-47b7-bfeb-82477e0c7b33 repair node keyspace done  5               abc                   0     2025-01-16T16:12:57Z 2025-01-16T16:12:59Z
+    1e028cb8-31a3-45ed-8728-af7a1ab586f6 repair node keyspace done  4               abc                   0     2025-01-16T16:12:45Z 2025-01-16T16:12:47Z
 
-    task_id                              type   kind scope    state sequence_number keyspace table entity
-    1e535f9b-97fa-4788-a956-8f3216a6ea8d repair node keyspace done  6               abc
-    d8926ee7-0faf-47b7-bfeb-82477e0c7b33 repair node keyspace done  5               abc
-    1e028cb8-31a3-45ed-8728-af7a1ab586f6 repair node keyspace done  4               abc
+    task_id                              type   kind scope    state sequence_number keyspace table entity shard start_time           end_time
+    1e535f9b-97fa-4788-a956-8f3216a6ea8d repair node keyspace done  6               abc                   0     2025-01-16T16:13:06Z 2025-01-16T16:13:09Z
+    d8926ee7-0faf-47b7-bfeb-82477e0c7b33 repair node keyspace done  5               abc                   0     2025-01-16T16:12:57Z 2025-01-16T16:12:59Z
+    1e028cb8-31a3-45ed-8728-af7a1ab586f6 repair node keyspace done  4               abc                   0     2025-01-16T16:12:45Z 2025-01-16T16:12:47Z
```

Comment:
Can we get an example with missing `end_time` and also one missing `start_time` too?

We should have tests with these cases too.

Reply:
Currently, for missing start_time/end_time API returns epoch and that's what is printed by nodetool.

I opened https://github.com/scylladb/scylladb/pull/22370 to change that to empty string, but I don't know if such a change is legal. If it is I will clean up the PR and open the issue. 

I'll wait for your opinion whether it can go in or it shouldn't before addressing the comment.

Reply:
It is fine for an API to return epoch, but user-facing tools should be friendlier .

Reply:
That said, please document that the relevant fields are set to epoch when there is no data.